src/components/DataTable/DataTable.js

```diff
@@ -0,0 +1,364 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import Button from '../Button';
+import Icon from '../Icon';
+import Search from '../Search';
+
+export class DataTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rows: props.initialRows,
```

Comment:
Do we need `initialRows` at all? It seems that the render function handles rendering entirely. This component doesn't even seem necessary.

Reply:
`<DataTable>` will eventually need to support a pagination use-case, and typically we start off with an initial page, or two, of data and then load additional pages when the user requests them. I think this work is just in preparation for handling that state logic 😄

Reply:
Hmm, thought the pagination use case was not being handled by this component at all, since it offloaded that work to the provided render method. I like that the datatable doesn't worry about rendering the rows, because than this component could handle infinite scrolling (using react-virtualized, or something) or go the traditional pagination route, without having to touch the implementation of the datatable.